activiti-cloud-starter-runtime-bundle/src/test/resources/application-test.properties

```diff
@@ -20,7 +20,15 @@ spring.cloud.stream.bindings.integrationEventsConsumer.contentType=application/j
 activiti.keycloak.test-user=testuser
 activiti.keycloak.test-password=password
 
+spring.cloud.stream.bindings.signalProducer.destination=signalEvent
+spring.cloud.stream.bindings.signalProducer.contentType=application/json
+spring.cloud.stream.bindings.signalConsumer.destination=signalEvent
+spring.cloud.stream.bindings.signalConsumer.group=mySignalConsumerGroup
+spring.cloud.stream.bindings.signalConsumer.contentType=application/json
+
+spring.activiti.asyncExecutorActivate=true
```

Comment:
@daisuke-yoshimoto what is this? do we need that?

Reply:
I mean.. the asyncExecutorActivate=true

Reply:
@Salaboy 
I fixed it. I removed this option since I change my impl to not use job executor.